symfony/framework-bundle/5.1/src/Kernel.php

```diff
@@ -15,14 +15,26 @@ protected function configureContainer(ContainerConfigurator $container): void
     {
         $container->import('../config/{packages}/*.yaml');
         $container->import('../config/{packages}/'.$this->environment.'/*.yaml');
-        $container->import('../config/{services}.yaml');
-        $container->import('../config/{services}_'.$this->environment.'.yaml');
+
+        if (file_exists(\dirname(__DIR__).'/config/services.yaml')) {
```

Comment:
I don't really understand this change.

Why not load it with a glob by default for the yaml file so it's loaded only if it exists, and load the php file if it exists?

Same comment for the `routes.php` vs `routes.yaml` file by the way.

Reply:
We must one XOR another. Loading both would create big WTF moments.

Reply:
Seems legit, I'm just unsure of how devs are going to figure out this new feature if they're really used to the yaml file 🤔

Seems similar to me than when the `.env` files were reorganized in the past